projects/components/src/data-exporter/data-exporter.component.html

```diff
@@ -1,5 +1,5 @@
 <clr-modal [clrModalOpen]="open" (clrModalOpenChange)="openChange.emit($event)" [clrModalSize]="'sm'" #modal>
-    <h3 class="modal-title">{{ dialogHeader || ('data-exporter.title' | translate) }}</h3>
+    <h3 class="modal-title">{{ dialogHeader || ('vcd.cc.data-exporter.title' | translate) }}</h3>
```

Comment:
Can we move the defaulting to the constructor?


```ts
 /**
   * Text for the export button.
   */
  @Input() exportText: string = this.translationService.translate('vcd.cc.cancel');

```

Here and in other usages